### PR TITLE
Office Add-ins Module 05 FY2018Q3 Content Refresh

### PR DESCRIPTION
**Pull request by andrewconnell**
Ran through lab ensuring everything works, screenshots are current and latest dependent packages are used.

- ran lab
- verified & updated where appropriate screenshots and instructions are current
- changed Angular & Vue projects to use explicit versions
- issue with Yeoman Office template unresolved in this lab... should be fixed when upstream repo resolved in https://github.com/OfficeDev/generator-office/pull/280

### CODE REVIEW

File: Teams/05 Microsoft Teams apps - Advanced Techniques/Lab.md [hunk 1]
```diff
@@ -336,11 +337,11 @@ Although not strictly necessary, in this lab the bot will be added to a new Team
...
 1. Enter a team name and description. In this example, the Team is named **teams-bot-1**. Click **Next**.
 1. Optionally, invite others from your organization to the team. This step can be skipped in this lab.
-1. The new team is shown. In the left-side panel, click the ellipses next to the team name. Choose **View team** from the context menu.
+1. The new team is shown. In the left-side panel, click the ellipses next to the team name. Choose **Manage team** from the context menu.
 
     ![](Images/Exercise1-12.png)
 
-1. On the View team display, click **Apps** in the tab strip. Then click the **Sideload an app** link at the bottom right corner of the application.
+1. On the Manage team display, click **Apps** in the tab strip. Then click the **Upload a custom app** link at the bottom right corner of the application.
 1. Select the zip file (**teams-bot1.zip** in this example) from the *bin* folder. Click **Open**.
```
Comment (kbrandl) [inline Teams/05 Microsoft Teams apps - Advanced Techniques/Lab.md]: For consistency, "bin" should be formatted as **bold** (not as *italic*).

File: Teams/04 Fundamentals of Microsoft Teams/Lab.md [hunk 2]
```diff
@@ -231,14 +233,14 @@ Before registering the bot, note the URL configured for the solution in Visual S
 
 ### Register the bot
 
-1. Go to the Microsoft Bot Framework portal at https://dev.botframework.com and sign in. (The bot registration portal accepts a Work or School Account or a Microsoft Account.)
-1. Click Register. (If the Register button is not shown, click **My bots** in the top navigation.)
+1. Go to the Microsoft Bot Framework Portal at https://dev.botframework.com/bots/new and sign in.
```
Comment (kbrandl) [inline Teams/04 Fundamentals of Microsoft Teams/Lab.md]: Other places in this PR where this same instruction occurs, it also includes this statement:  "(The bot registration portal accepts a Work or School Account or a Microsoft Account.)"   Should it be added here as well?

File: OfficeAddin/05 Using modern JavaScript/Demos/01 Office Add-in using React/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Office Add-ins: Build an Office Add-in using modern JavaScript tools and techniques - 300 Level
 
 ----------------
-In this demo, you will build an Office Add-in using a number of popular JavaScript tools and techniques. 
+In this demo, you will build an Office Add-in using React a popular JavaScript framework.
```
Comment (kbrandl) [inline OfficeAddin/05 Using modern JavaScript/Demos/01 Office Add-in using React/README.md]: Suggest adding a comma after the word "React" (i.e., "...using React, a popular JavaScript framework.")

File: OfficeAddin/05 Using modern JavaScript/Demos/02 Office Add-in using Angular/README.md [hunk 4]
```diff
@@ -1,13 +1,12 @@
 # Office Add-ins: Build an Office Add-in using modern JavaScript tools and techniques - 300 Level
 
-----------------
-In this demo, you will build an Office Add-in using a number of popular JavaScript tools and techniques. 
+In this demo, you will build an Office Add-in using Angular a popular JavaScript framework.
```
Comment (kbrandl) [inline OfficeAddin/05 Using modern JavaScript/Demos/02 Office Add-in using Angular/README.md]: Suggest adding a comma after the word "Angular" (i.e., "...using Angular, a popular JavaScript framework.")

File: OfficeAddin/05 Using modern JavaScript/Demos/03 Office Add-in using VueJS/README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Office Add-ins: Build an Office Add-in using modern JavaScript tools and techniques - 300 Level
 
 ----------------
-In this demo, you will build an Office Add-in using a number of popular JavaScript tools and techniques. 
+In this demo, you will build an Office Add-in using a VueJS a popular JavaScript framework.
```
Comment (kbrandl) [inline OfficeAddin/05 Using modern JavaScript/Demos/03 Office Add-in using VueJS/README.md]: Suggest adding a comma after the word "Vue" (i.e., "...using Vue, a popular JavaScript framework.")

File: OfficeAddin/README.md [hunk 6]
```diff
@@ -1 +1,9 @@
-# Office Addins
\ No newline at end of file
+# Office Addins
+
```
[the remark this answers is not in the document]
Comment (andrewconnell) [reply OfficeAddin/README.md]: Fixed.

File: OfficeAddin/05 Using modern JavaScript/Demos/01 Office Add-in using React/README.md (hunk 3 above)
Comment (andrewconnell) [reply OfficeAddin/05 Using modern JavaScript/Demos/01 Office Add-in using React/README.md]: Fixed

File: OfficeAddin/05 Using modern JavaScript/Demos/03 Office Add-in using VueJS/README.md (hunk 5 above)
Comment (andrewconnell) [reply OfficeAddin/05 Using modern JavaScript/Demos/03 Office Add-in using VueJS/README.md]: Fixed.

File: OfficeAddin/05 Using modern JavaScript/Demos/02 Office Add-in using Angular/README.md (hunk 4 above)
Comment (andrewconnell) [reply OfficeAddin/05 Using modern JavaScript/Demos/02 Office Add-in using Angular/README.md]: Fixed.

**Comment by andrewconnell** [comment]
Withdrawn... will resubmit

**Pull request closed without merging**